lib/utils/datahub.js

```diff
@@ -112,30 +113,91 @@ class DataHub extends EventEmitter {
     throw new Error(responseError(res))
   }
 
-  async pushFlow(specPath){
+  async pushFlow(specPath, dpJsonPath = '.datahub/datapackage.json'){
     let spec = {}
     let descriptor = {}
+
+    if (fs.existsSync(dpJsonPath)) {
+      let content
+      try {
+        content = fs.readFileSync(dpJsonPath)
+      } catch (err) {
+        throw new Error(err.message)
+      }
+      descriptor = JSON.parse(content)
+    }
+
     try {
```

Comment:
Same try-catch-throw overhead

Reply:
Maybe, we can handle error message saying `Couldn't read "flow.yaml". Please, check if it's correctly formatted.`